index.html

```diff
@@ -418,71 +417,21 @@
       </section>
       <section>
         <h3>
-          Push Service Discovery and API Use
+          Push API Use
```

Comment:
I think you lost some meaningful information in this heading. How about something like "Push service use" as what we really care about here is pointing to the IETF Web Push Protocol.